ts/src/bigone.ts

```diff
@@ -10,6 +10,10 @@ import { Int, OrderSide } from './base/types.js';
 
 //  ---------------------------------------------------------------------------
 
+/**
+ * @class bigone
```

Comment:
@pcriadoperez Is this correct jsdoc syntax?

In my PR I use 

```
@class
@name bigone
```

I also included a `@desc` but it looks like there's a `@classdesc` tag that might be more appropriate